frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -131,7 +131,8 @@
             </accessible-checkbox>
             <span>
               <a class="wp-table--details-link"
-                 ui-sref="work-packages.list.details.overview({workPackageId: row.object.id})">
+                 ui-state="desiredSplitViewState"
+                 ui-state-params="{workPackageId: row.object.id}">
```

Comment:
The `ui-state=` expression creates a watcher: https://github.com/angular-ui/ui-router/blob/master/src/ng1/directives/stateDirectives.ts#L207, but the params do not.

@romanroe: Can we provide an interpolated expression to `ui-sref`? Judging from the directive, it simply takes the args and parses that from string, so we might just do with `ui-sref="{{ desiredSplitViewState }}(...)` ?